backend/package.json

```diff
@@ -24,10 +24,17 @@
     "knex-script": "knex --knexfile",
     "knex": "./scripts/knex.sh",
     "migrate": "./scripts/migrate_latest.sh",
-    "migrate:all": "node ./scripts/createSqlDatabases.js && yarn migrate auth",
-    "migrate:make": "./scripts/migrate_make.sh && yarn seed:question-categories && yarn seed:question-difficulties",
+    "migrate:all": "yarn migrate:sql",
+    "migrate:make": "./scripts/migrate_make.sh",
+    "migrate:sql": "node ./scripts/createSqlDatabases.js && yarn migrate user",
+    "migrate-and-seed:all": "yarn migrate:all && yarn seed:all",
+    "migrate-and-seed:sql": "yarn migrate:sql && yarn seed:sql",
+    "seed:all": "yarn seed:sql && yarn seed:mongo",
     "seed:question-difficulties": "node ./scripts/seedQuestionDifficulties.js",
-    "seed:question-categories": "node ./scripts/seedQuestionCategories.js"
+    "seed:question-categories": "node ./scripts/seedQuestionCategories.js",
+    "seed:sql": "yarn seed:user",
+    "seed:mongo": "yarn seed:question-categories && yarn seed:question-difficulties",
```

Comment:
The mongo seeding is also very badly structured... Ideally, the logic should reside in each microservice, just like the one I did for knex